CMakeLists.txt

```diff
@@ -253,9 +290,11 @@ if(UNIX)
     endif(NATS_BUILD_NO_SPIN)
   endif(APPLE)
 endif(UNIX)
+
 add_definitions(-D${NATS_OS})
 add_definitions(-D_REENTRANT)
-if(NATS_BUILD_WITH_TLS)
+
+if(NATS_BUILD_WITH_TLS OR NATS_BUILD_WITH_WOLFSSL)
```

Comment:
Since I don't see where you would set `NATS_BUILD_WITH_TLS` to "false" when using WolfSSL, then not sure the "OR" is needed here, but we should also question if the following settings make sense if using WolfSSL library.